.github/workflows/rustsec.yml

```diff
@@ -0,0 +1,20 @@
+name: RustSec Audit Check
+
```

Comment:
You'll need to set the working directory

```yaml
defaults:
  run:
    working-directory: server
```

Reply:
good catch :finger_guns: